src/index.js

```diff
@@ -1,41 +1,47 @@
 const { spawn } = require("child_process");
 const fs = require("fs");
 const path = require("path");
-const {getBinaryName, normalize} = require("./utils.js");
+const { getBinaryName, normalize, getAuthInfo } = require("./utils.js");
+const constants = require("./constants.js");
+const WS = require("websocket").w3cwebsocket;
+const EventEmitter = require("events");
+
+class NeutralinoApp extends EventEmitter {
```

Comment:
Why do we need to extend `EventEmitter`?

Reply:
sir we need to extend the class with `EventEmitter` so that we can can support emitting events when a message is received from WS, i.e.
```js
        else if (message.event) {
          // Event from process
          if (message.event == "openedFile" && message.data.action == "dataBinary") {
            message.data.data = base64ToBytesArray(message.data.data);
          }
          this.emit(message.event, message.data);     // <--------
        }
```

by doing this, user can subscribe to events in backend, i.e.
```js
const app = new NeutralinoApp(...);
app.init();

app.on("windowBlur", ()=>{    // <-------
    console.log("window not in focus");
})
```

Reply:
Hello @viralgupta, thanks for your explanation. Please note that we need to match the original JavaScript client of Neutralinojs, so we need to use `app.events.on(.., ` to match `Neutralino.events.on`. I'll create an issue for this :tada: Thanks

Reply:
sure sir, we can continue this discussion on the issue created, thanks 😇

resolving conversation here.